src/app/shared/employement/tf-employement.component.html

```diff
@@ -0,0 +1,13 @@
+<div class="container">
+  <div class="panel panel-primary">
+    <div class="panel-heading">EmployementData information</div>
+    <div class="panel-body">
+      <div *ngIf="employementData">
+        <li>{{employementData.briefTeaser}}</li>
+        <br>
+        <li>{{employementData.slogan}}</li>
+        <br>
+      </div>
+    </div>
+  </div>
+</div>
```

Comment:
`tf-address.component.html` and this file has same html structure. In that case you should create a `shared component module` . and provide data to it.  so instead of two html's which have same structure you will have one html to maintain.